coalib/parsing/DefaultArgParser.py

```diff
@@ -189,6 +189,7 @@ def default_arg_parser(formatter_class=None):
 
     outputs_group.add_argument(
         '--filter-by', action='append', nargs='+',
+        metavar=('FILTER_NAME', 'FILTER_ARGS'),
```

Comment:
At least one filter arg has to be present.
So, output should be changed like this:
`[--filter-by FILTER_NAME [FILTER_ARGS ...]` -> `[--filter-by FILTER_NAME FILTER_ARG [FILTER_ARG ...]`